osu.Game.Modes.Taiko/Objects/TaikoHitObject.cs

```diff
@@ -44,7 +39,7 @@ public override void ApplyDefaults(TimingInfo timing, BeatmapDifficulty difficul
         {
             base.ApplyDefaults(timing, difficulty);
 
-            PreEmpt = BASE_SCROLL_TIME / difficulty.SliderMultiplier * timing.BeatLengthAt(StartTime) * timing.SpeedMultiplierAt(StartTime) / VelocityMultiplier / 1000;
+            ScrollTime = SCROLL_TIME / difficulty.SliderMultiplier * timing.BeatLengthAt(StartTime) * timing.SpeedMultiplierAt(StartTime) / 1000;

```

Comment:
This has to be wrong now. Since this was previously dividing by the VelocityMultiplier, whereas now it would effectively be multiplying by it. I believe the correct fix is to divide by SpeedMultiplierAt instead (scroll time should decrease as SpeedMultiplierAt increases).